### DISCUSSION TITLE
What do you recomend for local development?

### DISCUSSION THREAD

**Discussion by lucasmpr** [estuary/flow#1515] [Q&A]
We have an sql server that we want to stream some records to a postgres in near real time, estuary seems like a good fit for this use case.

The only problem is the local development workflow. Is it possible to spin this sync locally, so our devs can use the same system on their local machines? 

Is there something we can do? How People normally deal with this problem?

**Comment by psFried** [comment]
You can use `flowctl preview` to run any task locally. This is useful during initial development of the task, as it allows you to quickly change the `flow.yaml` file and re-run the `preview` to see what it does. And it can also be used to materialize your flow collections into a locally running database, if that's what you're after. `flowctl preview --help` has more details on options. Does that sound like what you're looking for?

**Comment by ton1517** [comment]
@psFried Hi. I'd like to know how to develop my application locally with estuary flow.
I'm trying to sync PostgreSQL data to Elasticsearch using Estuary Flow.
I can do it in dev environment, but I don't know how to build the same mechanism locally.
Is it possible to sync local PostgreSQL data to local Elasticsearch?
